centml/compiler/config.py

```diff
@@ -8,20 +9,19 @@ class CompilationStatus(Enum):
     DONE = "DONE"
 
 
-class Config:
+class Config(BaseSettings):
     TIMEOUT: int = 10
     MAX_RETRIES: int = 3
     COMPILING_SLEEP_TIME: int = 15
 
-    CACHE_PATH: str = os.getenv("CENTML_CACHE_DIR", default=os.path.expanduser("~/.cache/centml"))
+    CENTML_CACHE_DIR: str = "~/.cache/centml"
```

Comment:
we might need to use os.path.expanduser(). to resolve `~` into a full path. Probably helps if we want this to be OS agnostic.

Reply:
oh you're right. I meant to have it this way :facepalm: